package.json

```diff
@@ -10,9 +10,11 @@
     "i18next-browser-languagedetector": "^6.1.0",
     "lodash.clonedeep": "^4.5.0",
     "react": "^16.13.1",
+    "react-datepicker": "^4.0.0",
```

Comment:
FWIW, `<input type="date" />` will produce a native datepicker that is accessible, responsive and localized. Personally, I generally favour using native controls over libraries where possible - though it should be noted that often native controls may not be customizable/styleable.